app/services/ocrService.py

```
@@ -91,6 +91,18 @@ def download_from_gcs(gcs_path: str) -> bytes:
 
 
+def delete_from_gcs(gcs_path: str) -> bool:
+    try:
+        bucket = _get_bucket()
+        blob = bucket.blob(gcs_path)
+
+        blob.delete()
+        logger.info(f"GCS 삭제 완료: gs://{GCS_BUCKET_NAME}/{gcs_path}")
+    except Exception as e:
+        logger.exception("GCS 삭제 실패")
+        raise OcrError("GCS 삭제 중 오류가 발생했습니다.", is_client_error=False) from e
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)